index.js

```diff
@@ -88,7 +105,10 @@ console.log('hello world')
  * Example Output: "I called an arrow function!"
  */
 
-// TODO: ADD CODE HERE
+const myArrowFunction = () => {
+    return "This is a function expression!";
+}
```

Comment:
looks great!
Another way you can write the above, which you'll learn/the content probably hasn't covered yet is..
`const myArrowFunction = () => "This is a function expression!";`
^ You're allowed to write it like that when there's nothing else you need to do in the code block beside return. It's sort of a shorthand.